backend/src/main/java/com/htecgroup/skynest/model/response/FolderFileResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.htecgroup.skynest.model.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+public class FolderFileResponse {
```

Comment:
Why not renaming it to BucketResponse ?

Reply:
or BucketContentResponse because you already have the previous one

Reply:
it's just that getFolderContents will return the exact same response

if we call it BucketContentResponse, we can't really reuse it

Reply:
ok decide on the name, but `FolderFileResponse` to be honest doesn't tell me anything about the content of that object